cfgov/jinja2/v1/_includes/molecules/expandable.html

```diff
@@ -11,29 +11,29 @@
    label:                Label you want on the Expandable.
                          Default is an empty string.
 
-   settings
-   settings.is_bordered: Whether the Expandable has a bottom border or not.
+   value
+   value.is_bordered: Whether the Expandable has a bottom border or not.
                          Default is true.
-   settings.is_midtone:  Whether the Expandable is gray or not.
+   value.is_midtone:  Whether the Expandable is gray or not.
                          Default is false.
-   settings.is_expanded: Whether the Expandable is expanded or not.
+   value.is_expanded: Whether the Expandable is expanded or not.
                          Default is false.
    ========================================================================== #}
 
-{% macro render(label="", settings) %}
-{# Add the __expanded modifier class by default so all expandables are open
-   when JS is switched off, but add a data-js-hook to tell JS to remove it. #}
+{% if not value %}
```

Comment:
Would it be possible to break out the data prototyping from the JS route stuff and into a new PR? Doing both sorta breaks the "Changes are limited to a single goal (no scope creep)" rule